systemtests/src/main/java/io/enmasse/systemtest/platform/apps/SystemtestsKubernetesApps.java

```diff
@@ -97,6 +107,22 @@ public static String deployMessagingClientApp() throws Exception {
         return getMessagingAppPodName();
     }
 
+    public static String deployMessagingClientApp(String namespace) throws Exception {
+        if (!Kubernetes.getInstance().namespaceExists(namespace)) {
+            if (namespace.equals("allowed-namespace")) {
+                Namespace allowedNamespace = new NamespaceBuilder().withNewMetadata()
+                        .withName("allowed-namespace").addToLabels("allowed", "true").endMetadata().build();
+                Kubernetes.getInstance().getClient().namespaces().create(allowedNamespace);
+            } else {
+                Kubernetes.getInstance().createNamespace(namespace);
+            }
+
+        }
+        Kubernetes.getInstance().createDeploymentFromResource(namespace, getMessagingAppDeploymentResource(namespace));
+        TestUtils.waitForExpectedReadyPods(Kubernetes.getInstance(), namespace, 1, new TimeoutBudget(1, TimeUnit.MINUTES));
```

Comment:
I think 1 minute is a bit optimistic if you take into account the time to fetch images on a fresh cluster etc. Maybe bump it to 5.